src/table/utils/handle-accessibility.ts

```diff
@@ -1,11 +1,31 @@
-export const getCellElement = (rootElement, cellCoord) =>
+import { stardust } from '@nebula.js/stardust';
+import { BaseSyntheticEvent } from 'react';
+
+interface ICellFocusProps {
+  focusType: string;
+  cell: HTMLTableCellElement | undefined;
+}
+interface IHandleResetFocusProps {
+  focusedCellCoord: [number, number];
+  rootElement: HTMLDivElement;
+  shouldRefocus: {
+    current: boolean;
+  };
+  isSelectionMode: boolean;
+  setFocusedCellCoord: (coord: [number, number]) => void;
+  keyboard: stardust.Keyboard;
+  announce: (arg: { keys: string[] }) => void;
+}
+
+export const getCellElement = (rootElement: HTMLDivElement, cellCoord: [number, number]) =>
   rootElement.getElementsByClassName('sn-table-row')[cellCoord[0]]?.getElementsByClassName('sn-table-cell')[
     cellCoord[1]
-  ];
+  ] as HTMLTableCellElement;
 
-export const findCellWithTabStop = (rootElement) => rootElement.querySelector("td[tabindex='0'], th[tabindex='0']");
+export const findCellWithTabStop = (rootElement: Element) =>
```

Comment:
add return type

Reply:
Do you mean line 26 `as HTMLTableCellElement` ?

Reply:
@haxxmaxx I do not think it is necessary. It is implicitly set to HTMLTableCellElement already

Reply:
@Donya-qlik 
`export const findCellWithTabStop = (rootElement: Element): HTMLTableCellElement =>`

Reply:
ahh I did not know ts figures that out, nice. Then no need to change anything here :)